pulsar-client-admin/src/main/java/com/yahoo/pulsar/client/admin/internal/PersistentTopicsImpl.java

```diff
@@ -297,6 +299,38 @@ public void failed(Throwable throwable) {
     }
 
     @Override
+    public JsonObject getInternalInfo(String destination) throws PulsarAdminException {
+        try {
+            return getInternalInfoAsync(destination).get();
+        } catch (ExecutionException e) {
+            throw (PulsarAdminException) e.getCause();
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            throw new PulsarAdminException(e.getCause());
+        }
+    }
+
+    @Override
+    public CompletableFuture<JsonObject> getInternalInfoAsync(String destination) {
+        DestinationName ds = validateTopic(destination);
+        final CompletableFuture<JsonObject> future = new CompletableFuture<>();
+        asyncGetRequest(persistentTopics.path(ds.getNamespace()).path(ds.getEncodedLocalName()).path("internal-info"),
+                new InvocationCallback<String>() {
```

Comment:
if [server](https://github.com/merlimat/pulsar/blob/f8abc666619e7cbc11bb57f7e469e80ea9cfdf6a/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/PersistentTopics.java#L554) serialize `ManagedLedgerInfo` into json then if pass `InvocationCallback<ManagedLedgerInfo>` here : is JAXB not able to parse it?